_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,185 @@
+  - [Implementing an RFC](#implementing-rfcs)
+  - [RFC Postponement](#rfc-postponement)
+  - [Help this is all too informal!]
+  - [Attribution]
+
+# Core-Plans RFCs
+[Core-Plans RFCs]: #core-plans-rfcs
+
+A majority of contributor and maintainer work can be implemented and reviewed via [the normal GitHub pull request workflow](https://guides.github.com/introduction/flow/). These changes include bug fixes, documentation improvements, package version bumps, and adding a new core-plan. 
+
+Other changes by their nature are considered more complex, and we prefer to have an established process that these changes can be put through in order to provide the Habitat Maintainers and Community members those an opportunity to contribute to the design process and produce a consensus among the community.
+
+The "RFC" (request for comments) process is intended to provide a consistent path and expectation for a few things specifically: tooling changes, process changes, or new standards that might modify the acceptance criteria for new packages entering core-plans. The purpose is to ensure that all project stakeholders can be confident and aware of the direction core-plans and their maintenance are heading in.
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make any change considered "substantial" to the Core-Plans repository, tooling, or maintenance processes (including the RFC process itself). What constitutes a "substantial" change may evolve over time based on the community, but may initially include the following.
+
+  - Any process or operational changes to the Core-Plans Maintenance Policies.
+  - Any package feature or rewrite that breaks backwards compatibility for current users.
+  - Any Removal of a package's features (see breaking change).
+  - Changes to the standardized operational tooling in use by Core-Plans maintainers.
+  - Any changes to the standard core-plans accepted style and baseline.
+  - Any inbound Habitat change that obligates all entries in Core-Plans to change.
+
+**NOTE:** If you have opened a single PR that changes multiple plans, it is assumed you are making a substantial change to core plans. Substantial change PRs submitted without an RFC will be closed and you will be directed to open an RFC. If this was not your intention (for example in the case of updating two interdependent packages) please open separate PRs per package change.
+
+Some changes do not require an RFC:
+
+  - Package refactors/rephrases, changes that are transparent to user.
+  - Adding net "New" software to the core repository.
+  - Additions only likely to be _noticed by_ other maintainers (Invisible to target package users).
+  - Bug fixes for currently available packages.
+
+If you submit a pull request to implement a new feature without going through the RFC process, it may be closed with a polite request to submit an RFC first.
```

Comment:
Often an implementation can help bring clarity during the RFC process; perhaps a "Requires RFC" tag would be better than closing?

Reply:
Hmm. Perhaps we can do both? I'd prefer we don't leave PRs and issues hanging around for long periods of time. It might be too extreme a response for the size of the project but it's easier to define the process now rather than change it later I guess. We won't terminate the branch or anything, just close the PR, Leave a note, and include a label `S-Requires-RFC` ?